src/rad/handler.clj

```diff
@@ -138,11 +149,13 @@
       :description (:description page)
       :level level
     }
-  )
-)
+  ))
+
+(defn ms-from-epoch []
+  (.getTime (new java.util.Date)))
```

Comment:
```clojure
;; More idiomatic version of using new would be to just shorthand .
(.getTime (java.util.Date.))

;; and since we're in JVM, ever better would be
(System/currentTimeMillis)
```